server/middlewares/validateFunction.js

```diff
@@ -61,8 +139,15 @@ class Validator {
    * @returns {object} - status code and error message
    */
   getOrder(req, res, next) {
-    if (isNaN(req.params.id)) return res.status(400).send({ status: 'error', error: 'params shoudld be a number' });
-    if (!(parseInt(req.params.id, 10) === Number(req.params.id))) return res.status(400).send({ status: 'error', error: '' });
+    if (isNaN(req.params.id)) {
+      return res.status(400).send({
+        status: 'error',
+        error: 'id params shoudld be a number'
```

Comment:
I pointed out this typo before. It doesn't seem like you fixed it `shoudld` should be `should`.